packages/grid/data-grid/src/tests/toolbar.DataGrid.test.tsx

```diff
@@ -105,6 +115,11 @@ describe('<DataGrid /> - Toolbar', () => {
       expect(screen.getAllByRole('row')[1]).toHaveInlineStyle({
         maxHeight: `${Math.floor(rowHeight * COMPACT_DENSITY_FACTOR)}px`,
       });
+
+      // @ts-expect-error need to migrate helpers to TypeScript
+      expect(screen.getAllByRole('cell')[1]).toHaveInlineStyle({
```

Comment:
we could just load the maxHeight and do an equal assert to avoid the ts-error

Reply:
Yes but that tests the internal, not the visual part. For me this way the test is more secure. If we change the way the height of the cell is set then if we check the `maxHeight` the test will stop working, no?